core/src/test/java/org/pentaho/platform/engine/services/MockPluginManager.java

```diff
@@ -126,6 +126,11 @@ public Object getPluginSetting( String arg0, String arg1, String arg2 ) {
     return null;
   }
 
+  @Override
+  public boolean isCsrfProtectionEnabled(String pluginId) {
```

Comment:
Don't know why wingman didn't complain, but there should be spaces before and after `String pluginId`